pkg/controllers/workspace_controller.go

```diff
@@ -62,6 +68,18 @@ func (c *WorkspaceReconciler) Reconcile(ctx context.Context, req reconcile.Reque
 
 	klog.InfoS("Reconciling", "workspace", req.NamespacedName)
 
+	// Get the current API Server version.
```

Comment:
Guard this code with feature  gate?

Reply:
There is no harm from getting the k8s version with or without the feature gates.